### ISSUE TITLE
Unmet Dependency on .rmrr patcher with 6.8.0

### ISSUE THREAD

**Issue by 8064r7** [AnnabellaRenee87/Unraid-HP-Proliant-Edition#4]
Upgraded server today and when attempting the .rmrr patcher I noticed that the custom kernel didn't finish compiling due to a libffi unmet dependency.

"make: error while loading shared libraries: libffi.so.7: cannot open shared object file: No such file or directory"

unsure of workaround at this time & listing mostly to see if any others in the community are seeing this behavior as well at this time before I simply rollback to previous stable release and patch back on 6.7.

**Comment by CHBMB** [comment]
Yep, this is known and why they haven't released v6.8.0.  The patch that was being used is no longer working, so until someone figures out the patch, can't build it on Unraid
